calico-enterprise/reference/installation/tigerastatus.mdx

```diff
@@ -0,0 +1,62 @@
+---
+description: Descriptions of Tigera Status fields
+---
+
+# TigeraStatus
+
+Installing {{prodname}} on your Kubernetes cluster is managed by the Tigera operator. The operator get its configuration from the API resources and communicates through the Kubernetes API. The operator is deployed as a Deployment in the `tigera-operator` namespace, and records status in the `tigerastatus` resource. Depending on how your cluster is configured, you can get `tigerastatus` on these resources:
+
+- apiserver
+- applicationlayer
+- authentication
+- calico
+- calico-windows
+- compliance
+- egressgateway
+- intrusion-detection
+- log-collector
+- log-storage
+- manager
+- management-cluster-connection
+- policy-recommendation
+- secrets
```

Comment:
This list should include all of the log-storage-X names from below if we want to be comprehensive.

Reply:
Added log storage names to list.